### ISSUE TITLE
please add influx-cli to apt/yum repos

### ISSUE THREAD

**Issue by jdstrand** [influxdata/influx-cli#276]
The `influx-cli` package is currently listed in https://portal.influxdata.com/downloads/ but it lacks instructions for installing via apt/yum/dnf (indeed, the package is missing from https://repos.influxdata.com). It would be nice to have it available in the repos so users have an easier time staying up to date.

Thanks!

**Comment by danxmoran** [comment]
Blocked by #290
